src/main/java/frc/trigon/robot/commands/VisualizeNoteShootingCommand.java

```diff
@@ -20,7 +21,8 @@ public class VisualizeNoteShootingCommand extends Command {
     private Rotation2d startingPitch, startingYaw;
     private Translation2d initialXYVelocity;
     private double initialZVelocity;
-    private double noteZ;
+    private Pose3d notePose = new Pose3d();
+    private boolean hasCrossedFieldWidthBorder = false;
```

Comment:
Field width border just doesn't sit right with me, and not very self explanatory. 
Maybe `hasNoteCrossedAmpLine`? 
Anyways, I'd love some more amping videos, so satisfying and cool.